lib/core/src/lib/services/action.service.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+export interface Action {
+    type: string;
+    payload?: any;
+}
+
+@Injectable({ providedIn: 'root' })
+export class ActionService {
```

Comment:
Could you please explain why do we need it and how it should simplify redux usage?

Reply:
The applications that do not use Redux can still use the event bus service if needed... the NgRx library has a learning curve and not all the teams are using or willing to use that, the ADF has no dependencies on NgRx as well